core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -131,4 +140,36 @@
     text-align: center;
     text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
   }
+
+  simple-editor-body .question-type-wrapper {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
+    margin: 0 0 20px -30px;
+    overflow: hidden;
+    position: absolute;
+    width: 200px;
+    z-index: 20;
+  }
+  simple-editor-body .oppia-question-button {
```

Comment:
order the attributes like `background-color`, `border-color`, etc see our coding style [guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).